src/views/CreateType.vue

```diff
@@ -167,6 +164,23 @@ export default class Types extends Mixins(Translate) {
       },
     };
   }
+
+  public get dataTypeOptions(): Option[] {
```

Comment:
I think we should move the list of available data types to the `utils` or even an extra file!
I'm sure we'll add more data types later and it would be nice to have the definition at one place and not somewhere in a view :crystal_ball:

Reply:
done